hugo/content/docs/document-lifecycle.md

```diff
@@ -49,12 +49,12 @@ B --> C(<b>LangiumParser</b><br><br><i>parses a string into an AST</i>)
 
 Once all `LangiumDocument`s have been created, the `DocumentBuilder` service will sequentially process each `LangiumDocument` as described below.
 
-## Symbol Indexing
+## Indexing of Symbols
 Symbols are AST nodes that can be identified with a *name* and hence can be referenced from a *cross-reference*. Symbols that are *exported* can be referenced from other documents, while non-exported symbols are local to the document containing them. The `IndexManager` service keeps an index of all symbols that are exported from documents in the workspace. The set of all these exported symbols is called the *global scope*.
 
-Indexing of the exported symbols of an AST is executed on documents with the state `Parsed`. The default `ScopeComputation` service creates a list of `AstNodeDescriptions` for the root node (i.e. the node created by parsing the entry rule) and each named `AstNode` directly descending from the root node. Each `AstNodeDescription` object contains the `type` of the node, its identifier (i.e. the `name` property), the URI of the document where the node is located, and the location of the node inside of the document. The generated set of `AstNodeDescription`s makes symbols from a `LangiumDocument` available to other documents in the same workspace.
+Indexing of the exported symbols of an AST is executed on documents with the state `Parsed`. The default `AstNodeDescriptionProvider` service creates an `AstNodeDescription` for the root node (i.e. the node created by parsing the entry rule) and each named `AstNode` directly descending from the root node. This `AstNodeDescription` contains the `type` of the node, its identifier (i.e. the `name` property), the URI of the document where the node is located, and the location of the node inside of the document. The generated set of `AstNodeDescription`s makes symbols from a `LangiumDocument` accessible to other documents in the same workspace.
```

Comment:
Should the last sentence be revised to refer to the single AstNodeDescription rather than the set?